### PR TITLE
enable module fields overrides from custom beats

### PR DESCRIPTION
**Pull request by graphaelli**
follow up on #9188 where I unintentionally missed this originally - previously only libbeat fields could be overridden.

### CODE REVIEW

**Comment by ruflin** [comment]
Is this expected to change the fields.go files? Shouldn't the output stay the same?

**Comment by graphaelli** [comment]
the order of the fields changed, was: libbeat, custom, modules now: libbeat, modules, custom

File: libbeat/generator/fields/fields.go [hunk 1]
```diff
@@ -52,39 +52,45 @@ func NewYmlFile(path string, indent int) (*YmlFile, error) {
...
-	var err error
-	commonFields := []string{filepath.Join(esBeatsPath, "libbeat/_meta/fields.ecs.yml")}
-	if !isLibbeat(beatPath) {
-		commonFields = append(commonFields,
-			filepath.Join(esBeatsPath, "libbeat/_meta/fields.common.yml"),
-		)
-		libbeatModulesPath := filepath.Join(esBeatsPath, "libbeat/processors")
-		libbeatFieldFiles, err = CollectModuleFiles(libbeatModulesPath)
-		if err != nil {
+func makeYml(indent int, paths ...string) ([]*YmlFile, error) {
```
Comment (ruflin) [inline libbeat/generator/fields/fields.go]: +1 on extracting this.

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
retest

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

**Comment by ruflin** [comment]
@graphaelli Taking into account our current situation with CI I'm also happy with Travis only green.

**Pull request merged**